concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1779,7 +1779,11 @@ jobs:
                       PREFIX_JOB: "true"
                   EOF
 
-                  cf push
+                  # We only push if not running so stats are not disrupted
+                  # during deployments
+                  if ! cf app graphite-nozzle | grep started; then
```

Comment:
Multiple deployments will result in duplicate metrics AFAICT.

I realised that we wouldn't redeploy the graphite-nozzle, but figured that we touch it so infrequently a manual step to cause redeploy would be the lesser of the two evils - I just totally forgot to write that in the `What`. Sorry.

Ideally this should only re-deploy when the repo it builds from changes, but doing that in the current pipeline isn't feasible.